assets/src/edit-story/components/canvas/multiSelectionMovable.js

```diff
@@ -115,19 +115,21 @@ function MultiSelectionMovable( { selectedElements, nodesById } ) {
 	const onGroupEventEnd = ( { targets, isRotate, isResize } ) => {
 		targets.forEach( ( target, i ) => {
 			// Update position in all cases.
+			const frame = frames[ i ];
+			const [ editorWidth, editorHeight ] = frame.resize;
 			const properties = {
-				x: targetList[ i ].x + frames[ i ].translate[ 0 ],
-				y: targetList[ i ].y + frames[ i ].translate[ 1 ],
+				x: targetList[ i ].x + editorToDataX( frame.translate[ 0 ] ),
+				y: targetList[ i ].y + editorToDataY( frame.translate[ 1 ] ),
 			};
 			if ( isRotate ) {
-				properties.rotationAngle = frames[ i ].rotate;
+				properties.rotationAngle = frame.rotate;
 			}
-			if ( isResize ) {
-				properties.width = frames[ i ].resize.width;
-				properties.height = frames[ i ].resize.height;
+			if ( isResize && editorWidth !== 0 && editorHeight !== 0 ) {
```

Comment:
In which case would these be `0`?

Reply:
I synced this code from the `singleSelectionMovable`. These will be `0` where no actual resize happened and the sequence of events was simply: `onGroupResizeStart -> onGroupResizeEnd`. In other words, it's kind of an empty resize. Things would be a lot simpler if they simply gave us `deltaWidth/deltaHeight` as they do for position. If you want to try it on master, select two elements and quick click/release on a corner anchor. If you avoid any intermittent `onResize` events due to slight drag, the elements will jump to `0` size. That's what I wanted to avoid here.

If this keeps popping up, I think we may want to just switch to `deltaWidth/Height` ourselves.

Reply:
Ah, right, can remember now. Would you mind just leaving a comment for it or alternatively using something like
`const didResize = editorWidth !== 0 && editorHeight !== 0` and then using that instead?

Reply:
I like the `didResize` idea. Done.